<launch>
<node pkg="robo_vision" type="read_camera" name="read_usbcamera" output="screen">
    <param name="image_topic" value="/usbcamera/image"/>
    <param name="dev_name" value="/dev/ttyVideo0"/>
    <param name="image_width" value="640"/>
    <param name="image_height" value="480"/>
    <param name="exposure_time" value="63"/>
</node>
<node pkg="robo_vision" type="armor_detect" name="armor_detect" output="screen">
    <param name="intrinsic_file" value="/home/ubuntu/robot/src/robo_vision/param/camera-02-640.xml"/>
    <param name="offset_anlge_x" value="-2.0"/>
    <param name="offset_anlge_y" value="-0.7"/>
</node>
<!-- <node pkg="robo_vision" type="read_camera" name="read_fisheye_left" output="screen">
    <param name="image_topic" value="/fishcamera/left/image"/>
    <param name="dev_name" value="/dev/ttyVideoL"/>
    <param name="image_width" value="1280"/>
    <param name="image_height" value="720"/>
    <param name="exposure_time" value="64"/>
</node>
<node pkg="robo_vision" type="fishcamera_detect" name="fishcamera_detect" output="screen">
<param name="cam_left_center" value="656"/>
<param name="cam_left_radius" value="486"/>
<param name="cam_left_up" value="360"/>
<param name="cam_left_down" value="720"/>
<param name="cam_right_center" value="656"/>
<param name="cam_right_radius" value="486"/>
<param name="cam_right_up" value="0"/>
<param name="cam_right_down" value="720"/>
</node> -->
</launch>
